utils.py

```diff
@@ -198,5 +205,36 @@ def plot_mutations_per_site_along_seq(
         grid.ax_marg_y.remove()
         grid.fig.set_figwidth(20)
         grid.fig.set_figheight(8)
-        grid.ax_joint.set_xlabel("Position on genome")
+        grid.ax_joint.set_xlabel("Position on genome (Mb)")
         grid.ax_joint.set_ylabel("Number of mutations")
+
+    def plot_mutations_per_node(self, max_num_muts=None, show_counts=False):
```

Comment:
Would something like
```python
bins = None
if max_num_muts is not None:
    bins = range(max_num_muts + 1)
counts, edges, bars = plt.hist(
     self.nodes_num_mutations,
     bins=bins,
     # rest as before
)
```

Reply:
> Would something like this work?
> 
> ```python
> bins = None
> if max_num_muts is not None:
>     bins = range(max_num_muts + 1)
> counts, edges, bars = plt.hist(
>      self.nodes_num_mutations,
>      bins=bins,
>      # rest as before
> )
> ```

This works well, thanks. I'll work on nicer bin edge values later (i.e., when `bins=None`)

Reply:
Happy to merge!